app/controllers/grandparents_controller.rb

```diff
@@ -18,6 +18,14 @@ def create
     end
   end
 
+  def show
+    @grandparent = Grandparent.find(params[:id])
+    @description = params[:description]
```

Comment:
I don't think we need all the separate variables. These just be obtained within the view by accessing the @grandparent instance variable?
Something like `@grandparent.description`